gui/src/themes/stylekit.ts

```diff
@@ -44,6 +44,9 @@ export const stylekitTheme = {
     components: {
         // Form control
         MuiFormControl: {
+            defaultProps: {
```

Comment:
Should we then remove this ?

Reply:
I was wondering whether or not to leave it to cover potential use cases that I may not be aware of, but if you think I have every input covered with margin="dense" in those 3 components, I can remove it, yes.

Reply:
Done